pkgs/development/python-modules/digitalocean/default.nix

```
@@ -1,23 +1,44 @@
-{ stdenv, buildPythonPackage, fetchPypi, requests, jsonpickle }:
+{ buildPythonPackage
+, fetchFromGitHub
+, pythonPackages
```

for expressions in `python-modules`, please import the packages individually
```suggestion
, jsonpickle
, requests
, pytest
, responses
```

---

How come we do this btw? Easier to see what it depends on?

---

There's a lot of cases in applications where you want to change the version of a single version. Makes it easier to override individual dependencies.